patches/tModLoader/Terraria.ObjectData/TileObjectData.cs.patch

```diff
@@ -105,7 +105,7 @@
 +				// legacy compatibility, mismatch isn't a problem here
 +				if (Height < _tileObjectCoords.heights.Length && ModContent.LoadingMod.tModLoaderVersion < new Version(0, 11))
 +					return;
-+				
```

Comment:
Did [this](https://github.com/blushiemagic/tModLoader/pull/462/commits/dcc72bb1f7913a40b519e655e6c08212519396c0) fix it?

Reply:
Hmmm, probably shouldn't have done that using the online editor. *Probably*.

Reply:
No, chicken bones is talking about the removal of that newline

Reply:
oh

Reply:
I think I did break the setup. I can’t regen source anymore since I made that edit with the online editor. I’ll need a few more days before I’m at my keyboard again and can fix it, sorry ‘bout that.

Reply:
I think I fixed it with [bb8051a](https://github.com/blushiemagic/tModLoader/pull/462/commits/bb8051aab4631cee296dd50ce8e2192813dafc33)